UnityProject/Assets/Scripts/UI/Lobby/GUI_LobbyDialogue.cs

```diff
@@ -10,7 +10,7 @@ namespace Lobby
 {
 	public class GUI_LobbyDialogue : MonoBehaviour
 	{
-		private const string DefaultServerAddress = "localhost";
+		private const string DefaultServerAddress = "127.0.0.1";
```

Comment:
NOTE: Needs to use loopback address or some ping simulation tools won't work

Reply:
so `localhost` isn't recognized by some tools as a loopback address?

Reply:
yes, as I documented in the linked Wiki page

Reply:
er, maybe more accurate to just say it doesn't work with localhost, not sure whether that has to do with it being "recognized as a loopback address", more just that the tool doesn't seem to want to mess with things going over localhost but does as soon as you switch to 127.0.0.1